<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Othello</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="boardTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>45</y>
      <width>482</width>
      <height>482</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>482</width>
      <height>482</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>482</width>
      <height>482</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome To
Othello Game!!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="singlePlayBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>130</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hotseatPlayBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>130</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Hotseat Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="networkPlayBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>130</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Network Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreBlackLbl">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>넥슨Lv1고딕 Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scoreWhiteLbl">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>300</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>넥슨Lv1고딕 Bold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="blackLbl">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>넥슨Lv1고딕 Bold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="whiteLbl">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>200</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>넥슨Lv1고딕 Bold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>White</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>131</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
